src/main/java/bio/terra/cli/app/CommandRunner.java

```diff
@@ -85,6 +85,7 @@ public void runToolCommand(List<String> command, Map<String, String> envVars) {
 
     // add Terra global and workspace context information as environment variables
     Map<String, String> terraEnvVars = buildMapOfTerraReferences();
+    terraEnvVars.put("TERRA_USER_EMAIL", Context.requireUser().getEmail());
```

Comment:
// Keep in sync with notebook instance environment variables:
// https://github.com/DataBiosphere/terra-workspace-manager/blob/42a96e3efe78908d2969d1ac826cd84a11a16714/service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh#L165